src/components/SelectedProduct/SelectedProduct.tsx

```diff
@@ -1,9 +1,14 @@
-interface SelectedProps {
-  icon: string;
-}
+import BookMarkIcon from '../Icon/BookMarkIcon';
+import LikedIcon from '../Icon/LikedIcon';
+import styles from './selectedProduct.module.css';
 
-const SelectedProduct = (_: SelectedProps) => {
-  return <></>;
+const SelectedProduct = () => {
+  return (
```

Comment:
shorthand, does this component handle anything?

Reply:
fixed

Reply:
@thanhtrantruong47 Please answer `does this component handle anything`